t/native_cp_al/conf/native_cp_al_link_dest.conf.in

```diff
@@ -0,0 +1,7 @@
+config_version	1.2
+snapshot_root	@SNAP@
+cmd_rsync		@RSYNC@
+sync_first	1
+link_dest	1
+interval		hourly	6
+backup			@TEMP@	native_cp_al/
```

Comment:
And this / should be missing too I think

Reply:
Just going to rm that test completely. See other comment about it.